src/backend/adapters/base.ts

```diff
@@ -117,5 +117,5 @@ export abstract class BaseAdapter {
 
   abstract serializeComponent(el: any, event: string): TreeNode;
 
-  abstract cleanup(): void;
+  // abstract cleanup(): void;
```

Comment:
let's stay away from keeping commented out code, we have history with source control and can always go back if needed. I would remove it.